packages/trpc/server/routers/viewer/organizations/adminVerify.handler.ts

```diff
@@ -57,5 +57,69 @@ export const adminVerifyHandler = async ({ input }: AdminVerifyOptions) => {
     },
   });
 
-  return { ok: true, message: "Verified Organization" };
+  const foundUsersWithMatchingEmailDomain = await prisma.user.findMany({
+    where: {
+      email: {
+        endsWith: acceptedEmailDomain,
+      },
+      teams: {
+        some: {
+          teamId: input.orgId,
+        },
+      },
+    },
+    select: {
+      id: true,
+      email: true,
+    },
+  });
```

Comment:
Find only users with this email domain that have a membership with this email domain

Reply:
We could make this Distinct on userId not sure if its needed tho

Reply:
What happens if there's no found users?

Reply:
All the above below should just not have any effect - we can early return tho i guess